src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+// 1: отримання елементу з html;
+
+const element = document.querySelectorAll('span.population');
```

Comment:
Consider checking if the elements are found before proceeding with further operations. If `document.querySelectorAll('span.population')` returns an empty NodeList, subsequent operations may not behave as expected.